packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/VarForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import { Element, FormField, Input } from '@codesandbox/components';
+
+export const VarForm = props => {
+  const [name, setName] = useState(props.name || '');
+  const [value, setValue] = useState(props.value || '');
+
+  const submit = e => {
+    e.preventDefault();
+
+    if (name && value) {
+      props.onSubmit({
+        name,
+        value,
+      });
+      setName('');
+      setValue('');
+    }
+  };
+
+  const onKeyDown = e => {
+    if (e.key === 'Escape' && props.onEscClicked) {
```

Comment:
Should we just use a form and then skip the `escape` implementation? 🤔

Reply:
I involved danny and we decided to do both, esc still stays for more advanced people but we have the button too :)